apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -108,8 +110,25 @@ const InnerFileBrowser = React.memo(
 
     const handleDeleteFolder = (folderId: string) => {
       const folderName = folders[folderId].name;
+      const projectFolders = Object.values(folders);
+      const projectFiles = Object.values(files);
+      const folderCount = findSubFolders(folderId, projectFolders).length;
+      const fileCount = findFiles(
+        folderId,
+        projectFiles,
+        projectFolders
+      ).length;
+
       const title = `Are you sure?`;
-      const message = `Are you sure you want to delete the folder ${folderName}? This will delete all files and folders inside ${folderName}.`;
+      let message = `Are you sure you want to delete the folder "${folderName}"?`;
```

Comment:
I know this wasn't previously translated, but would you mind putting the new strings in [this translation file](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/codebridge/en_us.json)? It's already imported in this component as `codebridgeI18n`. If you wanted to add "Are you sure?" as well that would be great 😄

Reply:
Yup, I can take care of that on Monday!

Reply:
Updated. I also added a couple other stragglers that were in this same file.